src/pybind/mgr/dashboard_v2/controllers/rgw.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import re
+import json
+import cherrypy
+
+from ..tools import ApiController, RESTController
+from .. import logger
+from mgr_module import MgrModule, CommandResult
+
+
+@ApiController('rgw')
+class Rgw(RESTController):
```

Comment:
You are not making use of the `RESTController` functionality in your code, so you can just extend from `BaseController`.

Reply:
This will change in the future. There's ongoing migration from oA to `dashboard_v2`. It's about the RGW proxy functionality.